lib/matplotlib/axes/_axes.py

```diff
@@ -5211,47 +5200,50 @@ def pcolor(self, *args, **kwargs):
-            An :class:`matplotlib.colors.Normalize` instance is used
-            to scale luminance data to 0,1. If *None*, defaults to
-            :func:`normalize`.
+            pcolor can be very slow for large arrays; consider
+            using the similar but much faster
+            :func:`~matplotlib.pyplot.pcolormesh` instead.
 
-          *vmin*/*vmax*: [ *None* | scalar ]
-            *vmin* and *vmax* are used in conjunction with *norm* to
-            normalize luminance data.  If either is *None*, it
+        Parameters
+        ----------
+        C : array_like
+            An array of color values.
+        X : array_like, optional
+            X coordinates of the colored quadrilaterals.
+        Y : array_like, optional
+            Y coordinates of the colored quadrilaterals.
+        cmap : `~matplotlib.colors.Colormap`, optional, default: None
+            If `None`, default to rc settings.
+        norm :`matplotlib.colors.Normalize`, optional, default: None 
+            An instance is used to scale luminance data to 0,1. 
+            If `None`, defaults to :func:`normalize`.
+        vmin, vmax : scalar, optional, default: None
+            `vmin` and `vmax` are used in conjunction with `norm` to
+            normalize luminance data.  If either is `None`, it
             is autoscaled to the respective min or max
-            of the color array *C*.  If not *None*, *vmin* or
-            *vmax* passed in here override any pre-existing values
-            supplied in the *norm* instance.
-
-          *shading*: [ 'flat' | 'faceted' ]
+            of the color array `C`.  If not `None`, `vmin` or
+            `vmax` passed in here override any pre-existing values
+            supplied in the `norm` instance.
+        shading : {'flat', 'faceted'}, optional, default: 'flat'
             If 'faceted', a black grid is drawn around each rectangle; if
             'flat', edges are not drawn. Default is 'flat', contrary to
             MATLAB.
-
             This kwarg is deprecated; please use 'edgecolors' instead:
-              * shading='flat' -- edgecolors='none'
-              * shading='faceted  -- edgecolors='k'
-
-          *edgecolors*: [ *None* | ``'none'`` | color | color sequence]
-            If *None*, the rc setting is used by default.
-
+              `shading`='flat' -- `edgecolors`='none'
+              `shading`='faceted'  -- `edgecolors`='k'
+        edgecolors : {None, ``'none'``, color, color sequence}
+            If `None`, the rc setting is used by default.
             If ``'none'``, edges will not be visible.
-
             An mpl color or sequence of colors will set the edge color
-
-          *alpha*: ``0 <= scalar <= 1``   or *None*
-            the alpha blending value
-
-          *snap*: bool
+        alpha : scalar, optional, default: None
+            The alpha blending value, between 0 (transparent) and 1 (opaque).
+        snap: bool, optional, default: TODO
```

Comment:
Don't forget to add a space in between snap and column, or else the documentation won't be rendered properly.